apps/services/user-notification/src/app/modules/notifications/notifications.controller.ts

```diff
@@ -105,6 +105,8 @@ export class NotificationsController {
       messageId: id,
       ...flattenedArgs,
       ...body,
+      args: {}, // Remove args, since they're in a better format in `flattenedArgs`
+      queue: { url: this.queue.url, name: this.queue.queueName },
```

Comment:
not sure i understand the value of this, wont this simply log the same queue url and name per environment for millions of messages ?

Reply:
The intent is to log the queue _name_, which is semi-related to the URL. Also, the URL is only available post-startup, while the name should be available on class instantiation.

This make it easier to filter by the human-readable queue name, which should be the same on any environment. The queue URL (I think), changes per environment, acting as an ID.

Reply:
But yes, this does increase the log volume, but comparatively, it's minor. I'd prefer better searching/filtering over a few more cents going to the Clouds.